src/libraries/System.Text.Json/tests/Utf8JsonReaderTests.TryGet.Date.cs

```diff
@@ -150,7 +150,7 @@ public static void TestingStringsInvalidConversionToDateTimeOffset(string jsonSt
 
         [Fact]
         // https://github.com/dotnet/runtime/issues/30095.
```

Comment:
nit: unrelated to the change, but remove the `.`
```suggestion
        // https://github.com/dotnet/runtime/issues/30095
```